### PR TITLE
chore: Update packages to the latest patches

### PR DESCRIPTION
**Pull request by lipis**
https://github.com/tjunnone/npm-check-updates

running the

```
ncu -u --semverLevel minor
```

### CODE REVIEW

**Comment by lipis** [comment]
What do you think about installing the https://dependabot.com to be always up to date with the dependencies?

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "node": ">=4"
   },
   "dependencies": {
-    "babel-code-frame": "7.0.0-alpha.12",
-    "babylon": "7.0.0-beta.33",
+    "babel-code-frame": "7.0.0-beta.3",
+    "babylon": "7.0.0-beta.34",
```
Comment (azz) [inline package.json]: FYI babylon beta versions can have breaking changes

Comment (suchipi) [reply package.json]: Based on your approval, is this one safe?

Comment (lipis) [reply package.json]: let me know if I need to downgrade this one..

Comment (existentialism) [reply package.json]: It is safe, we didn't make any changes to babylon 33->34 besides updating our flow version internally.

**Pull request merged**